README.md

```diff
@@ -64,6 +64,21 @@ At boot time, you might need `adb shell setenforce 0` for B2G to boot (flash_sar
   
 If need to output a zip ROM file, you can use `./build.sh dist DIST_DIR=dist_output` instead of `./build.sh` in step 5.
 
+## Generic B2G System Images (GSI)
+
+1. Fetch the code: `REPO_INIT_FLAGS="--depth=1" ./config.sh gsi_b2g`
+2. Setup your environment to fetch the custom NDK: `export LOCAL_NDK_BASE_URL='ftp://ftp.kaiostech.com/ndk/android-ndk'`
+3. Install Gecko dependencies: `cd gecko && ./mach bootstrap`, choose option 4 (Android Geckoview).
+4. Apply patch: `./patcher/patcher.sh`
+5. Build: `./build-gsi.sh [type name] systemimage`
+
+   type:
+   - `gsi_arm64_ab` (arm64 partitionA/B)
+   - `gsi_arm64_a` (arm64 partitionA)
```

Comment:
```
FAILED: ninja: unknown target 'gsi_arm64_a'
```

Reply:
I need to check `build-gsi.sh`, there should be something wrong

Reply:
did you try `build-gsi.sh gsi_arm64_ab systemimage`

Reply:
Maybe the pack of patching?

Reply:
yes ...
```
  PATH=$HOME/node_modules/.bin/:$PATH ./build-gsi.sh gsi_arm64_ab systemimage
```

Failing the same way after patching